### DISCUSSION TITLE
Legion GO - Quit LSDaemon problem

### DISCUSSION THREAD

**Discussion by drunksnake** [Valkirie/HandheldCompanion#894] [Q&A] [answered]
Hi,
I am using handheld companion for the Legion GO since first release with the Patreon, but I have the same notifications since the begining and I can't find a way to fix them.

The first one tells me that I should qui LSDaemon so that the application can run properly and without any errors, but each time i close it, it opens again instantly. Anyone know how to solve this ?

Second one I got is HWiNFO shared memory limit was reached. Again, I can't find a way to fix this.

Thank you,

**Comment by MSeys** [comment]
How do you close LSDaemon? Via Task Manager?

For HWiNFO, you need to enable that option within HWinfo itself as explained.

**Reply by drunksnake** [reply]
I tried both task manager and the button directly in the notifications of handheld companion. For HWiNFO i found out how to enable It.

**Reply by MSeys** [reply, marked as answer]
You need to disable DAService in Services. That's the only correct way.

**Comment by jmen1213** [comment]
I believe there is a new service called DAService. If you go into Services and disable it, the error goes away. However, for me, HC can no longer detect the physical controller. Anyone else experiencing this on the Legion go?

**Reply by MSeys** [reply]
@jmen1213 We have released a hotfix build to fix that, yesterday. Please update.
